src/views/Export.vue

```diff
@@ -1,92 +1,168 @@
+              @input="edited = true"
+            />
+          </v-col>
+        </v-form>
+      </v-row>
+
+      <v-row>
+        <v-col>
+          <p>Click "Download" to export data as an Excel spreadsheet.</p>
+          <v-btn color="primary" @click="download" :loading="downloading">
+            <v-icon left>file_download</v-icon>Download
+          </v-btn>
+        </v-col>
+      </v-row>
+      <div id="error">{{ error }}</div>
+      <p :hidden="!downloading">Download is in progress...</p>
+    </v-col>
+  </v-container>
 </template>
 
 <script lang="ts">
-import Component from "vue-class-component";
-import AppCard from "@/components/AppCard.vue";
-import ActionButtonGroup from "@/components/ActionButtonGroup.vue";
-import BoxedButton from "@/components/BoxedButton.vue";
 import Vue from "vue";
+import Component from "vue-class-component";
 import * as firebase from "firebase/app";
 import "firebase/functions";
 import "firebase/storage";
 import moment from "moment";
+// components
+import DatePickerMenu from "@/components/Export/DatePickerMenu.vue";
+import DateQuickSelect from "@/components/Export/DateQuickSelect.vue";
 
 @Component({
   components: {
-    ActionButtonGroup,
-    AppCard,
-    BoxedButton,
+    DatePickerMenu,
+    DateQuickSelect,
   },
 })
 export default class Settings extends Vue {
-  active = false;
   downloading = false;
-  end_date = moment().format("YYYY-MM-DD");
-  start_date = moment().subtract(6, "days").format("YYYY-MM-DD");
+
+  startDate = moment().startOf("week").format("YYYY-MM-DD");
+
+  endDate = moment().format("YYYY-MM-DD");
+
   error = "";
 
-  handler() {
+  edited = false;
+
+  /**
+   * Navigates to home page
+   */
+  goHome(): void {
     this.$router.push({
       name: "home",
     });
   }
 
-  download() {
+  /**
+   * Sets `startDate` and `endDate` based on preset intervals
+   */
+  quickSelect(selection: string): void {
+    this.edited = false;
+    if (selection === "thisWeek") {
+      this.startDate = moment().startOf("week").format("YYYY-MM-DD");
+      this.endDate = moment().format("YYYY-MM-DD");
+    }
+    if (selection === "lastWeek") {
+      const startOfLast = moment().startOf("week").subtract(1, "weeks");
+      this.startDate = startOfLast.format("YYYY-MM-DD");
+      this.endDate = startOfLast.add(6, "day").format("YYYY-MM-DD");
+    }
+    if (selection === "weekToDate") {
+      this.startDate = moment().subtract(7, "days").format("YYYY-MM-DD");
+      this.endDate = moment().format("YYYY-MM-DD");
+    }
+    if (selection === "monthToDate") {
+      this.startDate = moment().subtract(1, "months").format("YYYY-MM-DD");
+      this.endDate = moment().format("YYYY-MM-DD");
+    }
+    if (selection === "prevMonth") {
+      this.startDate = moment()
+        .subtract(1, "months")
+        .startOf("month")
+        .format("YYYY-MM-DD");
+      this.endDate = moment()
+        .subtract(1, "months")
+        .endOf("month")
+        .format("YYYY-MM-DD");
```

Comment:
consistency with `startOf`/`endOf` and `subtract`/`add` (vs. line 107, `DateQuickSelect.vue` lines 36-38)